ydb/core/blobstorage/nodewarden/node_warden_vdisk.cpp

```diff
@@ -175,6 +175,7 @@ namespace NKikimr::NStorage {
         vdiskConfig->ReplPausedAtStart = Cfg->VDiskReplPausedAtStart;
         vdiskConfig->EnableVPatch = EnableVPatch;
         vdiskConfig->FeatureFlags = Cfg->FeatureFlags;
+        vdiskConfig->Apply(Cfg->VDiskPerformanceConfig);
```

Comment:
It's better to obtain this from StorageConfig. Which is already updated dynamically, by the way. Or at least supposed to.